src/parsers.js

```diff
@@ -10,6 +10,7 @@ function parseQueryStringInput (queryString) {
   var overheads = ['fee', 'tax']; // tip is a special case, it maps to tipPercent
 
   var pairs = queryString.split('&');
+  pairs = pairs.map(pair => decodeURIComponent(pair).trim());
```

Comment:
Shouldn't the `decodeURIComponent` be around `pairValues[0]` below (line 24) in order to be consistent with the encoding?